persistence/migrations/migrations.go

```diff
@@ -1,9 +1,13 @@
 package migrations
 
 import (
+	"sync"
+
 	"github.com/go-pg/migrations/v8"
 )
 
+var sqlDiscoveryMtx sync.Mutex
+
 var Collection = migrations.NewCollection()
```

Comment:
That's nothing but use
var (

)

To group these declarations

Reply:
I've added the grouping syntax, but reduced the number of variables here back to 2. Should I still group, and should I keep an empty line between the two variable declarations?

Reply:
Looks good (that was only a little note!)